src/jobs/ios_build_and_test.yml

```diff
@@ -90,6 +94,7 @@ steps:
       steps:
         - metro_start
   - ios_build:
+      pod_install_directory: <<parameters.pod_install_directory>>
```

Comment:
As per my other comment, can you instead use the `pod_install` command directly instead of making it part of `ios_build`, but only if the `pod_install_directory` parameter is set?